src/content/docs/apm/agents/nodejs-agent/hosting-services/nodejs-agent-microsoft-azure.mdx

```diff
@@ -4,12 +4,59 @@ tags:
   - Agents
   - Nodejs agent
   - Hosting services
+  - Azure
+  - Azure Site Extension
 metaDescription: How to install APM for Node.js on Microsoft Azure Web Apps.
 redirects:
   - /docs/agents/nodejs-agent/hosting-services/nodejs-agent-microsoft-azure
+  - /docs/agents/nodejs-agent/installation-configuration/install-nodejs-agent-azure-site-extension
   - /docs/nodejs/nodejs-agent-on-microsoft-azure
+translate:
+  - jp
+  - kr
 freshnessValidatedDate: never
 ---
+## Easy Install with the New Relic Azure Site Extension for Node [#site-extension-install]
+
+You can install the Node.js agent onto your Azure-managed web apps with the New Relic Azure Site Extension for Node. This doc walks you through compatibility, installation, and configuration to get your Azure web app data into New Relic.
+
+## Compatibility and requirements [#compatibility-requirements]
+
+Azure Site Extensions are only available for Windows-based App Service apps that are deployed as code. For applications running on Windows 32, full Code Level Metrics support (file path, line, column) is not available. Profiling falls back to function name only.
+
+## Install the Node.js agent with Azure Site Extension
+
+To install the New Relic Azure Site Extension, add the `NewRelic.Azure.WebSites.Extension.NodeAgent` site extension from the Azure home page. Follow these steps:
+
+1. From the Azure Home page, click the App Services tile, then select the target application in the displayed list.
+2. Scroll down to <DNT>**Extensions**</DNT> in the options listed on the left under the <DNT>**Development Tools**</DNT> category.
+3. Click on <DNT>**+ Add**</DNT> at the top of the page to toggle the extension drop down, then select <DNT>**New Relic Node Agent**</DNT>. Check the box for accepting the legal terms
+4. Click <DNT>**Add**</DNT> on the bottom of the page to start installing the extension.
+
+Once installed, the extension creates the following artifacts:
+
+- Folder: `C:\home\SiteExtensions\NewRelic.Azure.Websites.Extension.NodeAgent`
+- XDT: `applicationHost.xdt` that will add the necessary `NODE_OPTIONS` environment variable on application startup
```

Comment:
```suggestion
* XDT: `applicationHost.xdt` that will add the necessary `NODE_OPTIONS` environment variable on application startup
```

Reply:
We usually spell out acronyms on first use, and we did this patterrn in some of the other agent set ups. What do you think?

Reply:
I'd generally be all for it, though people working with Microsoft systems and XML probably know it as XML Document Transform, so I'm not sure if we need to define it explicitly.